core/internal/cltest/cltest.go

```diff
@@ -1683,3 +1687,49 @@ func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
 	err = tx.Commit()
 	require.NoError(t, err)
 }
+
+// FlagSetApplyFromAction applies the flags from action to the flagSet.
+// `parentCommand` will filter the app commands and only applies the flags if the command/subcommand has a parent with that name, if left empty no filtering is done
+func FlagSetApplyFromAction(action interface{}, flagSet *flag.FlagSet, parentCommand string) {
+	cliApp := cmd.Client{}
+	app := cmd.NewApp(&cliApp)
+
+	foundName := parentCommand == ""
+	actionFunctionName := getFunctionName(action)
```

Comment:
nit/ for readability: s/`Function`/`Func` and adding to `action`:
```suggestion
	actionFuncName := getFuncName(actionFunc)
```

Reply:
Fixed in [44845a3](https://github.com/smartcontractkit/chainlink/pull/8173/commits/44845a3d1de9a6ac3def557c0763f53451e4f980)